contrib/bindings/go/pathrs/capi.go

```diff
@@ -0,0 +1,178 @@
+// Copyright (C) 2019, 2020 Aleksa Sarai <cyphar@cyphar.com>
```

Comment:
My comment for this entire file is that I'm not sure it's worth having separate code for the C bindings. Personally I think it'd read more reasonably if everything was in one file (rather than adding additional wrapping). But I could be wrong...